.gitignore

```diff
@@ -18,6 +18,10 @@ build/
 typesversions
 .cache
 .yarnrc
+
+# Ignore package-lock.json from example projects
+examples/query/react/**/package-lock.json
+
```

Comment:
```suggestion
```

This will not be necessary in master any more since we switched to yarn workspaces - there is only one lock file in the repo root now :)